k8s/deploy_app.sh

```diff
@@ -1,20 +1,17 @@
 #!/usr/bin/env bash
-
-set -e
+set -euxo pipefail
 
 if [[ -z "$SUBMISSION_BUCKET_NAME" ]]; then
   echo "SUBMISSION_BUCKET_NAME is mandatory"
   exit 1
 fi
 
-DOCKER_REGISTRY="${DOCKER_REGISTRY:-eu.gcr.io/census-eq-ci}"
-IMAGE_TAG="${IMAGE_TAG:-latest}"
-REQUESTED_CPU_PER_POD="${REQUESTED_CPU_PER_POD:-3}"
-ROLLING_UPDATE_MAX_UNAVAILABLE="${ROLLING_UPDATE_MAX_UNAVAILABLE:-1}"
-ROLLING_UPDATE_MAX_SURGE="${ROLLING_UPDATE_MAX_SURGE:-1}"
-MIN_REPLICAS="${MIN_REPLICAS:-3}"
-MAX_REPLICAS="${MAX_REPLICAS:-10}"
-TARGET_CPU_UTILIZATION_PERCENTAGE="${TARGET_CPU_UTILIZATION_PERCENTAGE:-50}"
+gcloud auth activate-service-account --key-file "${GOOGLE_APPLICATION_CREDENTIALS}"
```

Comment:
I think the `activate-service-account` command should be alongside the writing of the credentials file in the task. Using a service account is how Concourse deploys the app, but if running locally someone could use their user credentials.